### PR TITLE
Feature/create request uri payload

### PR DESCRIPTION
**Pull request by JanDeDobbeleer**

### CODE REVIEW

File: dotnet-core/Client.cs [hunk 1]
```diff
@@ -8,7 +7,7 @@ public class Client
     {
         public Client()
         {
-            Environment.CurrentDirectory = Directory.GetCurrentDirectory();
+            System.Environment.CurrentDirectory = Directory.GetCurrentDirectory();
```
Comment (RoelandMatthijssens) [inline dotnet-core/Client.cs]: I don't really understand the need for this change. Can you elaborate a bit more?

Comment (RoelandMatthijssens) [reply dotnet-core/Client.cs]: Ah, your next commit makes it clear.
I would squash the two commits.

File: demos/dotnet-core/Integrations/ItsmeClient.cs [hunk 2]
```diff
@@ -16,22 +18,48 @@ public ItsmeClient(IConfiguration config)
...
-            urlSettings.Scopes = new string[]{"profile", "email", "address", "phone"};
-            urlSettings.RequestUri = "https://example.com:443/production/request_uri";
+            urlSettings.ServiceCode = "BMIDITP_LOGIN";
+            urlSettings.Scopes = new List<Itsme.Scope>(){
+                Itsme.Scope.profile,
+                Itsme.Scope.email,
+                Itsme.Scope.address,
+                Itsme.Scope.phone
+            };
+            urlSettings.RequestUri = "https://hbkbizj26f.execute-api.eu-central-1.amazonaws.com:443/production/request_uri";
```
Comment (RoelandMatthijssens) [inline demos/dotnet-core/Integrations/ItsmeClient.cs]: Shouldn't we make this configurable with env vars?

File: demos/golang/main.go [hunk 3]
```diff
@@ -71,6 +71,33 @@ func handleRedirect(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprint(w, string(userJSON))
 }
 
+func handleRequest(w http.ResponseWriter, r *http.Request) {
```
Comment (RoelandMatthijssens) [inline demos/golang/main.go]: This chance is out of scope of the commit message. I would put this in a separate commit

File: dotnet-core/Client.cs (hunk 1 above)
Comment (JanDeDobbeleer) [reply dotnet-core/Client.cs]: Will do.

File: demos/dotnet-core/Integrations/ItsmeClient.cs (hunk 2 above)
Comment (JanDeDobbeleer) [reply demos/dotnet-core/Integrations/ItsmeClient.cs]: The URL should change, but no.  As this is a dynamic URL (no need for it to be identical as you can have multiple depending on the actual request, it's added like this for illustrative purposes.

File: demos/golang/main.go (hunk 3 above)
Comment (JanDeDobbeleer) [reply demos/golang/main.go]: I'll check, not sure what changed exactly.

Comment (JanDeDobbeleer) [reply demos/golang/main.go]: Done. (need to push)
